src/generator/writers/writeStdlib.ts

```diff
@@ -1915,12 +1919,15 @@ export function writeStdlib(ctx: WriterContext) {
 
     ctx.fun(`__tact_log2`, () => {
         ctx.signature(`int __tact_log2(int num)`);
+        ctx.flag("impure");
         ctx.context("stdlib");
         ctx.asm(`asm "DUP 5 THROWIFNOT UBITSIZE DEC"`);
     });
 
     ctx.fun(`__tact_log`, () => {
         ctx.signature(`int __tact_log(int num, int base)`);
+        ctx.flag("impure");
+        ctx.flag("inline");
```

Comment:
and if we want to add `inline` modifiers I think we should run some benchmarks and compare the gas usage with `inline_ref`.

Reply:
`inline_ref` isn't used widely in Tact stdlib so I guess we can merge it with this `inline` modifier and maybe do a benchmarking on most of the stdlib functions later